docs/core/tutorials/index.md

```diff
@@ -6,13 +6,13 @@ ms.date: 03/16/2017
 titleSuffix: ""
 ms.custom: seodec18
 ---
-# Learn .NET Core and the .NET Core SDK tools by exploring these Tutorials
+# Learn .NET Core and the .NET Core SDK tools by exploring these tutorials
 
 The following tutorials are available for learning about .NET Core.
 
-## Building applications with Visual Studio 2017
+## Create applications with Visual Studio
```

Comment:
I like that this is now version independent. That's why I didn't make a comment on the metadata for this file.